sklearn/ensemble/tests/test_voting.py

```diff
@@ -125,6 +139,35 @@ def test_weights_iris():
     assert_almost_equal(scores.mean(), 0.93, decimal=2)
 
 
+def test_weights_regressor():
+    """Check weighted average regression prediction on boston dataset."""
+    reg1 = DummyRegressor(strategy='mean')
+    reg2 = DummyRegressor(strategy='median')
+    reg3 = DummyRegressor(strategy='quantile', quantile=.2)
+    ereg = VotingRegressor([('mean', reg1), ('median', reg2),
+                            ('quantile', reg3)], weights=[1, 2, 10])
+
+    X_r_train, X_r_test, y_r_train, y_r_test = \
+        train_test_split(X_r, y_r, test_size=.25)
+
+    reg1_y = reg1.fit(X_r_train, y_r_train).predict(X_r_test)
+    reg2_y = reg2.fit(X_r_train, y_r_train).predict(X_r_test)
+    reg3_y = reg3.fit(X_r_train, y_r_train).predict(X_r_test)
+    ereg_y = ereg.fit(X_r_train, y_r_train).predict(X_r_test)
+
+    avg = np.average(np.asarray([reg1_y, reg2_y, reg3_y]), axis=0,
+                     weights=[1, 2, 10])
+    assert_almost_equal(ereg_y, avg, decimal=2)
+
+    ereg_n = VotingRegressor([('mean', reg1), ('median', reg2),
```

Comment:
what do `n` and `o` stand for?

maybe rename into `ereg_weights_none` and `ereg_weight_equal`